newm-server/src/main/kotlin/io/newm/server/features/cardano/repo/CardanoRepositoryImpl.kt

```diff
@@ -302,6 +304,35 @@ internal class CardanoRepositoryImpl(
         )
     }
 
+    override suspend fun getWalletMusicNFTs(xpubKey: String): List<List<LedgerAssetMetadataItem>> {
```

Comment:
Does it make sense to make this a single list? Or perhaps have a `Map<String, List<LedgerAssetMetadataItem>>>` where the string is a differentiator between each list. (policy ID maybe?)

Reply:
@cristhianescobar  It would have to be something like `<policy_id>.<asset_name>`. Policy by itself isn't unique. I'll discuss on slack for other options...